modules/exploits/multi/http/cockpit_cms_rce.rb

```diff
@@ -183,22 +183,23 @@ def report_cred(opts)
   def login(un, pass)
     print_status('Attempting login')
     res = send_request_cgi(
-      'uri' => '/auth/login'
+      'uri' => '/auth/login',
+      'keep_cookies' => true
     )
+
     fail_with(Failure::UnexpectedReply, "#{peer} - Could not connect to the web service") unless res
     fail_with(Failure::UnexpectedReply, "#{peer} - Could not connect to the web service") unless /csfr\s+:\s+"([^"]+)"/ =~ res.body
-    cookie = res.get_cookies
-    res = send_request_raw(
+
+    res = send_request_cgi(
       'uri' => '/auth/check',
       'method' => 'POST',
       'ctype' => 'application/json',
-      'cookie' => cookie,
```

Comment:
I think I'm missing something here, this was previously sending a cookie but now it's not? I guess the cookie is being kept from the _previous_ request so you don't need to specify anything here do I have that right?

Reply:
Previous code extracted the cookie from an initial request and passed it explicitly. New code tells the initial request to `keep_cookies`, which handles all that automatically.

Reply:
yea that makes sense, it just looks counter intuitive to be removing the cookie from the request here, cheers

Reply:
For explicitness it might be nice to provide `keep_cookies` anyway? 👀

Reply:
Agreed

Reply:
This has been marked as resolved, but it like this is still missing the `keep_cookies => true` option? 👀